File: src/main/java/org/cru/godtools/fragment/AboutFragment.java
package org.cru.godtools.fragment;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import org.keynote.godtools.android.R;

public class AboutFragment extends Fragment {
    @NonNull
    public static Fragment newInstance() {
        return new AboutFragment();
    }

    /* BEGIN lifecycle */

    @Override
    public View onCreateView(@NonNull final LayoutInflater inflater, @NonNull final ViewGroup container,
                             @Nullable final Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_about, container, false);
    }

    /* END lifecycle */
}

File: app/src/main/java/org/cru/godtools/fragment/AboutFragment.kt
package org.cru.godtools.fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import org.cru.godtools.R

fun createAboutFragment(): Fragment = AboutFragment()

class AboutFragment : Fragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_about, container, false)
    }
}
